sonar-update-center-common/src/test/java/org/sonar/updatecenter/common/PluginReferentialTest.java

```diff
@@ -102,6 +103,22 @@ public void should_add_dependency() {
     assertThat(pluginReferential.findPlugin("bar").getRelease(Version.create("1.0")).getOutgoingDependencies().iterator().next().getVersion().getName()).isEqualTo("1.0");
   }
 
+
+  @Test
+  public void ignore_add_license_dependency() {
+    Plugin licensePlugin = Plugin.factory("license");
+    Release licenseRelease = new Release(licensePlugin, "1.0");
+    licensePlugin.addRelease(licenseRelease);
+
+    Plugin bar = Plugin.factory("bar");
+    Release bar10 = new Release(bar, "1.0");
+    bar.addRelease(bar10);
```

Comment:
nitpick: what about factoring the creation of a plugin with a single release:

```
  private static Plugin createPluginWithSingleRelease(String pluginName) {
    Plugin licensePlugin = Plugin.factory(pluginName);
    Release licenseRelease = new Release(licensePlugin, "1.0");
    licensePlugin.addRelease(licenseRelease);
    return licensePlugin;
  }
```

On top of those 2 places, it would improve the readability of several other test cases

Reply:
I gave it a try, and actually, I don't find it _that_ useful. A lot of tests depends both on the `Plugin` and `Release`, which makes this `createPluginWithSingleRelease` method actually usable only in a couple of places. I prefer to leave it as it is now.